Kickstarter-iOS/Tests/ViewModels/UpdatePreviewViewModelTests.swift

```diff
@@ -143,7 +143,7 @@ final class UpdatePreviewViewModelTests: TestCase {
       |> UpdateDraft.lens.update.id .~ 1
       |> UpdateDraft.lens.update.projectId .~ project.id
 
-    let api = MockService(fetchProjectResponse: project, publishUpdateError: .couldNotParseJSON)
+    let api = MockService(publishUpdateError: .couldNotParseJSON, fetchProjectResponse: project)
```

Comment:
the changed order is the order in `MockService` and unfortunately `MockService`'s variables has never been properly alphabetized :/